src/screens/SettingsScreen.tsx

```diff
@@ -293,6 +293,51 @@ function ViewerFontFamilyPreferenceSelector() {
   );
 }
 
+function ViewModePreferenceSelector() {
+  const theme = useTheme();
+  const dispatch = useDispatch();
+  const [selectViewModeOpen, setViewModePreferenceOpen] = React.useState(false);
+  const viewSettings = useSelector((state: StoreState) => state.settings.viewSettings);
+  const { defaultViewMode = "last" } = viewSettings;
+
+  const prettyName = {
+    last: "Last used",
+    viewer: "Preview",
+    editor: "Editor",
+  };
+
+  return (
+    <List.Item
+      title="Default View Mode"
+      description="Set the default view mode for notes"
+      right={(props) =>
+        <Select
+          {...props}
+          visible={selectViewModeOpen}
+          onDismiss={() => setViewModePreferenceOpen(false)}
+          options={["last", "viewer", "editor"]}
+          titleAccossor={(x) => prettyName[x]}
+          onChange={(selected) => {
+            setViewModePreferenceOpen(false);
+            if (selected === defaultViewMode) {
+              return;
+            }
+            dispatch(setSettings({
+              viewSettings: {
+                ...viewSettings,
+                defaultViewMode: selected as typeof defaultViewMode,
```

Comment:
This cast shouldn't be needed. Maybe it's because of how you did the options which is untyped. What you probably want is to add to the top of this function something like:
```
const options: ViewModeKey[] = ["last", "viewer", "editor"];
```